### PR TITLE
HBW-315 Customize fonts in HBW

### PR DESCRIPTION
**Pull request by NatalieKor**

### CODE REVIEW

File: hbw/app/assets/stylesheets/hbw/brand_variables.sass [hunk 1]
```diff
@@ -65,3 +65,184 @@
...
+  // Process name
+  --base-hbw-process-name-font-family: var(--base-hbw-default-font-family)
+  --base-hbw-process-name-font-size: 1.5em
+  --base-hbw-process-name-font-weight: normal
+  --base-hbw-process-name-font-style: normal
+
+  // BP tabs
+  --base-hbw-nav-tabs-font-family: var(--base-hbw-default-font-family)
+  --base-hbw-nav-tabs-font-size: var(--base-font-size)
+  --base-hbw-nav-tabs-font-weight: 400
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/brand_variables.sass:87]: It's normal.

File: hbw/app/assets/stylesheets/hbw/brand_variables.sass [hunk 2]
```diff
@@ -65,3 +65,184 @@
   --base-select-dropdown-indicator-color-hover: #333
   --base-select-dropdown-indicator-color-focus: #666
   --base-select-dropdown-indicator-width: 36px
+
+  //Fonts 
+  --base-hbw-default-font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif  
+
+  //Panel title
+  --base-hbw-panel-title-font-family: var(--base-hbw-default-font-family)
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/brand_variables.sass:73]: IMO we should keep the same family through all elements, don't turn widget into patchwork.

**Review by TimAle — CHANGES_REQUESTED** [review]

File: hbw/app/assets/stylesheets/hbw/brand_variables.sass (hunk 1 above)
Comment (NatalieKor) [reply hbw/app/assets/stylesheets/hbw/brand_variables.sass:87]: Screenshot 2021-04-01 at 11 28 02

File: hbw/app/assets/stylesheets/hbw/branding.sass [hunk 3]
```diff
@@ -47,8 +47,23 @@ $button-types: default primary danger info success warning
   &::-webkit-scrollbar-thumb:hover
     background: var(--hbw-webkit-scrollbar-thumb-hover-color, var(--base-hbw-webkit-scrollbar-thumb-hover-color))
 
+.select-table-label
+  font-family: var(--hbw-font-family, var(--base-hbw-font-family))
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/branding.sass:51]: Let's calculate this value once.

File: hbw/app/assets/stylesheets/hbw/brand_variables.sass [hunk 4]
```diff
@@ -65,3 +65,155 @@
...
+  --base-hbw-nav-tabs-font-size: var(--base-font-size)
+  --base-hbw-nav-tabs-font-weight: normal
+  --base-hbw-nav-tabs-font-style: normal
+
+  // String input
+  --base-hbw-form-group-input-font-size: var(--base-font-size)
+  --base-hbw-form-group-input-font-weight: normal
+  --base-hbw-form-group-input-font-style: normal
+
+  // Attechers text
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw/brand_variables.sass:92]: Typo.

**Review by TimAle — CHANGES_REQUESTED** [review]

**Review by TimAle — DISMISSED** [review]

**Review by TimAle — APPROVED** [review]

**Pull request merged**